bedrock/firefox/tests/test_firefox_details.py

```diff
@@ -399,6 +399,58 @@ def test_latest_major_version_no_int(self):
         """latest_major_version should return 0 when no int."""
         eq_(firefox_desktop.latest_major_version('release'), 0)
 
+    @override_settings(FUNNELCAKE_LOCALES='en-US', FUNNELCAKE_PLATFORMS='win')
```

Comment:
These should always be lists, not strings. The `cast=Csv()` ensures that the values of these settings will be lists, even if they're single element. But using `@override_settings` means that these won't pass through the cast function.